Functions_Lab/script.js

```diff
@@ -0,0 +1,33 @@
+console.log("hello world")
+
+
+/*
+Write a function named getAreaOfCircle. It takes a radius parameter. It calculates and returns the area of a circle with that radius.
+Write a function named getCircumferenceOfCircle. It takes a radius parameter. It calculates and returns the circumference of a circle with that radius.
+Write a function named getAreaOfSquare. It takes a side parameter. It calculates and returns the area of a square with that side length.
+Write a function named getAreaOfTriangle. It takes two parameters: base and height. It calculates and returns the area of a triangle with that base and height.
+*/
+
+
+const getAreaOfCircle = (radius) => {
+    return Math.PI * radius * radius 
+}
+    console.log(getAreaOfCircle(6))
```

Comment:
Look out for consistent indentation

```suggestion
const getAreaOfCircle = (radius) => {
    return Math.PI * radius * radius 
}

console.log(getAreaOfCircle(6))
```